### ISSUE TITLE
X# compiler doesn't understand ENDSEQUENCE (dialect XBase++)

### ISSUE THREAD

**Issue by DenGhostYY** [X-Sharp/XSharpPublic#1242]
**Describe the bug**
X# compiler doesn't understand ENDSEQUENCE. In XSharpDefs.xh no `#command ENDSEQUENCE => end`

**To Reproduce**
```
procedure main()
    local x
    local cMsg
    
    begin sequence
        x := 0

        if x == 0
            break "Error"
        endif

    recover using cMsg
        ? cMsg
    endsequence

    ? x - 1
    return
```

**Expected behavior (xBase++ exe)**
Output
```
Error
-1
```

**Actual behavior (X# compiler)**
```
Error	XS1003	Syntax error, 'END SEQUENCE' expected
```

**Additional context**
X# Compiler version 2.14.0.4 (release)
xsc.exe -dialect:xBase++ -xpp1 -lb -memvar -vo1 -vo5 -vo10 -vo15 -vo16

**Comment by RobertvanderHulst** [comment]
I am translating `ENDSEQUENCE `to `END SEQUENCE` to make sure that the following code throws an error (which it does not in Xbase++, because ENDSEQUENCE is translated to END)

```
procedure main()
if true
   ? "Hello world"
ENDSEQUENCE
wait
return
```

**Comment by RobertvanderHulst** [comment]
Fixed in f6f92f349540c50e5a11eda11eea1cc3ad81b792

**Comment by cpyrgas** [comment]
Confirmed fixed
